HanabiTracker/Views/CurrentHand.swift

```diff
@@ -4,9 +4,28 @@ import Model
 struct CurrentHand: View {
     @EnvironmentObject var hand: Hand
     @Environment(\.options) private var options: Binding<Options>
+    @AppStorage("showsConventions") private var showsConventions = true
```

Comment:
My thought was that maybe I should use this in the other place and keep it out of options entirely?